pkg/log/log_messages.go

```diff
@@ -80,3 +80,17 @@ const CAKC064 string = "CAKC064 Unable to find Authenticator for this configurat
 const CAKC065 string = "CAKC065 File path is invalid %s"
 const CAKC066 string = "CAKC066 Performing authn-jwt"
 const CAKC067 string = "CAKC067 Failed reading jwt token from %s"
+const CAKC068 string = "CAKC068 Loading config object..."
+const CAKC069 string = "CAKC069 Picking configuration object..."
+const CAKC070 string = "CAKC070 Chosen %q configuration"
+const CAKC071 string = "CAKC071 Gathering input settings for configuration..."
+const CAKC072 string = "CAKC072 Gathered input settings successfully"
+const CAKC073 string = "CAKC073 Validating input configuration..."
+const CAKC074 string = "CAKC074 Validated input configuration successfully"
+const CAKC075 string = "CAKC075 Chosen %q authenticator"
+const CAKC076 string = "CAKC076 Loading JWT token from %s"
+const CAKC077 string = "CAKC077 Loaded JWT token successfully"
+const CAKC078 string = "CAKC078 Extracting host from the configuration"
+const CAKC079 string = "CAKC079 Extracted identity %s from configuration"
+const CAKC080 string = "CAKC080 No host in the configuration. Identity will be provided in the token"
```

Comment:
No host was detected in authn configuration. Host identity will be taken from the JWT provided in the request.

@tzheleznyak @shulifink  WDYT?